quickstart1/app/Http/Controllers/QuickStartController.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\Task;
+use Illuminate\Http\Request;
+use Illuminate\Database;
+class QuickStartController extends Controller
+{
+    public function index()
+    {
+        $tasks = Task::all() ;
+         return view( 'QuickStart_view' , compact( 'tasks' ));
```

Comment:
1. chừa 1 dòng trắng trên return chứ không phải 1 space em nhé 
2. tên file view thì dùng chữ thường chữ k phải chữ in hoa 
3. sửa lại thế này `view('index', compact('tasks'));`
Em đọc coding convention để viết cho chuẩn này
> https://github.com/framgia/coding-standards/tree/master/vn/php